Testreglar/3.3.4/App/app-3.3.4b.json

```diff
@@ -0,0 +1,224 @@
+			"oblig": true,
+			"ruting": {
+				"alle": {
+					"type": "gaaTil",
+					"steg": "3.2"
+				}
+			}
+		},
+		{
+			"stegnr": "3.2",
+			"spm": "Prøv å slette brukerstyrt data.",
+			"ht": "<p>Slett lagret, brukerstyrt data.</p>\n<p><strong>Merk: </strong>Dersom det ikke er data på appsiden fra før, må du registrere og lagre data før du kan gjennomføre testing.</p>",
+			"type": "instruksjon",
+			"ruting": {
+				"alle": {
+					"type": "gaaTil",
+					"steg": "3.3"
+				}
+			}
+		},
+		{
+			"stegnr": "3.3",
+			"spm": "Har appsiden en mekanisme for å bekrefte at du har til hensikt å slette dataen?",
+			"ht": "<p>Mekanismen for å bekrefte, skal informere brukeren om handlingen (slettingen) du er i ferd med å gjennomføre og hvilke konsekvenser det får.</p>\n<p>Mekanismen kan for eksempel være:</p>\n<ul>\n<li>En avkryssingsboks, knapp eller lignende for å bekrefte eller avbryte sletting</li>\n<li>Ledeteksten til mekanismen kan for eksempel være \"Jeg bekrefter sletting\", \"Sikker på at du vil slette informasjonen?\" eller \"Bekreft sletting\"</li>\n</ul>",
+			"type": "jaNei",
+			"kilde": [
+				"G155",
+				"G168"
+			],
+			"ruting": {
+				"ja": {
+					"type": "gaaTil",
+					"steg": "3.4"
+				},
+				"nei": {
+					"type": "gaaTil",
+					"steg": "3.6",
+					"delutfall": {
+						"nr": 1,
+						"fasit": "Nei",
+						"tekst": "Appsiden har ikke en mekanisme for å bekrefte sletting av brykerstyrte data."
+					}
+				}
+			}
+		},
+		{
+			"stegnr": "3.4",
+			"spm": "Er mekanismen for å bekrefte sletting, separat frå funksjonaliteten for å fullføre sletting?",
+			"ht": "<p>Mekanismen for å bekrefte og funksjonaliteten for å fullføre sletting, skal være to separate skjemaelement.</p>\n<p>Slik tester du:</p>\n<ul>\n<li>Sjekk at mekanismen for å bekrefte sletting, er separat fra funksjonen for å fullføre sletting.</li>\n</ul>\n<p><strong>Merk:</strong> En kombinasjon av mekanisme og funksjonen for å fullføre sletting, er ikke godkjent. Eksempelvis er en knapp «Bekreft og slett» er ikke tilstrekkelig.</p>",
+			"type": "jaNei",
+			"kilde": [
+				"G155"
+			],
+			"ruting": {
+				"ja": {
+					"type": "gaaTil",
+					"steg": "3.5"
+				},
+				"nei": {
+					"type": "gaaTil",
+					"steg": "3.6",
+					"nr": 1,
+					"delutfall": {
+						"nr": 1,
+						"fasit": "Nei",
+						"tekst": "Appsiden har en mekanisme for å bekrefte sletting av brukterstyrt data. Mekanismen er ikke separat fra funksjonaliteten for å fullføre sletting."
+					}
+				}
+			}
+		},
+		{
+			"stegnr": "3.5",
+			"spm": "Er mekanismen for å bekrefte sletting av data obligatorisk?",
+			"ht": "<p>Mekanismen for å bekrefte sletting av data, skal være et obligatorisk skjemaelement.</p>\n<p>Slik tester du:</p>\n<ul>\n<li>Prøv å sette data, uten å bruke mekanismen for å bekrefte.</li>\n<li><strong>Merk:</strong> Hvis du ikke får slettet data, uten å krysse av eller tilsvarende, betyr det at mekanismen fungerer som den skal (er obligatorisk).</li>\n</ul>",
```

Comment:
Kunne godt ta med hvordan vi vurder ja/nei knapper som dukker opp i en dialogboks, av typen vil du slette? [Ja] [Nei]. Faller disse under obligatorisk skjemaelement?

Reply:
@Sivzilla82 Kan du lage et forslag på dette?

Reply:
Det blir for spesifikk. Hvis brukeren skal svare for å komme videre, så er det åpenbart obligatorisk.